tools/toolchains/__init__.py

```diff
@@ -1531,24 +1564,22 @@ def report(self):
         to_ret.update(self.config.report)
         return to_ret
 
-from tools.settings import ARM_PATH
-from tools.settings import GCC_ARM_PATH
-from tools.settings import IAR_PATH
+from tools.settings import ARM_PATH, ARMC6_PATH, GCC_ARM_PATH, IAR_PATH
 
 TOOLCHAIN_PATHS = {
     'ARM': ARM_PATH,
-    'uARM': ARM_PATH,
+    'ARMC6': ARMC6_PATH,
     'GCC_ARM': GCC_ARM_PATH,
     'IAR': IAR_PATH
 }
 
-from tools.toolchains.arm import ARM_STD, ARM_MICRO
+from tools.toolchains.arm import ARM_STD, ARMC6
 from tools.toolchains.gcc import GCC_ARM
 from tools.toolchains.iar import IAR
 
 TOOLCHAIN_CLASSES = {
     'ARM': ARM_STD,
-    'uARM': ARM_MICRO,
+    'ARMC6': ARMC6,
```

Comment:
are we removing uARM ,because ARM_MICRO class is still defined ? Is this breaking change or what is the intention here? or I have missed something - most probably